vizro-core/examples/visual-vocabulary/pages/_factories.py

```diff
@@ -110,3 +110,42 @@ def column_and_line_factory(group: str):
             make_code_clipboard_from_py_file("column_and_line.py"),
         ],
     )
+
+
+def waterfall_factory(group: str):
+    """Reusable function to create the page content for the column chart with a unique ID."""
+    return vm.Page(
+        id=f"{group}-waterfall",
+        path=f"{group}/waterfall",
+        title="Waterfall",
+        layout=vm.Layout(grid=PAGE_GRID),
+        components=[
+            vm.Card(
+                text="""
+
+                #### What is a waterfall chart?
+
+                A waterfall chart is a bar chart that shows the cumulative effect of sequential positive or negative
+                values. It starts with an initial value, displays individual changes as steps, and ends with the final total.
```

Comment:
If you get more linting errors regarding the lines being too long, you can just break off the sentences in a new line :)

```suggestion
                values. It starts with an initial value, displays individual changes as steps, and ends with the 
                final total.
```

Reply:
Awesome! I was working on fixing linting errors, and realized you finished them off! Thank you :)

Reply:
You did amazing already! I thought I quickly finish it off, so you don't have to iterate a third time 😄 It's merged! 🥳